jobs/anonymizeConferences/index.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../../lib/db")
+
+module.exports = async () => {
+  console.debug("Start of anonymisation job")
```

Comment:
on peut ptet enlever les logs, puisqu'ils sont spammy

Reply:
> J'ai testé en local.
> 
> Pour la review app, j'ai créé des confs, et je propose qu'on attende lundi pour voir si elles ont bien été nettoyées.
> (j'ai starté un container de cron sur scalingo, par defaut les review apps n'ont pas de container de cron donc pas de cron)
> id 	expiresAt
> 4cf8bc25-*** 	2020-12-04 22:59:00+00
> b895f340-*** 	2020-12-07 22:59:00+00
> 2286a13f-*** 	2020-12-15 22:59:00+00

```
audioconf_t_8385=> select * from conferences;
                  id                  |                email                |  phoneNumber  | durationInMinutes |           createdAt           |       expiresAt        | canceledAt | domain
--------------------------------------+-------------------------------------+---------------+-------------------+-------------------------------+------------------------+------------+--------
 4cf8bc25-70ee-4101-ada9-c8b6c14b3e2d | estelle.comment@beta.gouv.fr        | 0033972155000 |                   | 2020-12-04 16:21:35.601052+00 | 2020-12-04 22:59:00+00 |            |
 b895f340-2e9b-40a0-beae-2b2829fae58b | estelle.comment@beta.gouv.fr        | 0033972155000 |                   | 2020-12-04 16:23:39.73377+00  | 2020-12-07 22:59:00+00 |            |
 2286a13f-4534-4551-96e1-522e130b5153 | estelle.comment@beta.gouv.fr        | 0033972155000 |                   | 2020-12-04 16:23:46.513694+00 | 2020-12-15 22:59:00+00 |            |
 a4264225-9ada-4c5e-90ae-5f73de095c3c | pierre-olivier.mauguet@beta.gouv.fr | 0033972155000 |                   | 2020-12-07 14:26:11.512887+00 | 2020-12-14 22:59:00+00 |            |
(4 rows)

audioconf_t_8385=> select * from conferences;
                  id                  |                email                |  phoneNumber  | durationInMinutes |           createdAt           |       expiresAt        | canceledAt |    domain
--------------------------------------+-------------------------------------+---------------+-------------------+-------------------------------+------------------------+------------+--------------
 b895f340-2e9b-40a0-beae-2b2829fae58b | estelle.comment@beta.gouv.fr        | 0033972155000 |                   | 2020-12-04 16:23:39.73377+00  | 2020-12-07 22:59:00+00 |            |
 2286a13f-4534-4551-96e1-522e130b5153 | estelle.comment@beta.gouv.fr        | 0033972155000 |                   | 2020-12-04 16:23:46.513694+00 | 2020-12-15 22:59:00+00 |            |
 a4264225-9ada-4c5e-90ae-5f73de095c3c | pierre-olivier.mauguet@beta.gouv.fr | 0033972155000 |                   | 2020-12-07 14:26:11.512887+00 | 2020-12-14 22:59:00+00 |            |
 4cf8bc25-70ee-4101-ada9-c8b6c14b3e2d |                                     | 0033972155000 |                   | 2020-12-04 16:21:35.601052+00 | 2020-12-04 22:59:00+00 |            | beta.gouv.fr
(4 rows)
```

Seems good to me @estellecomment

Reply:
Ils sont spammy surtout pour les stats (2 par minute). Pour l'anonymization, ce sera 1x par jour. Moins problématique.